src/ECharts/Utils.purs

```diff
@@ -193,139 +162,91 @@ numeralFormatterWithValMnplt add mul template =
   
   where
   converter_0 :: Number -> String
-  converter_0 num = show $ Int.round ((num * 1.0 + add) * mul)
+  converter_0 = precision 0 
```

Comment:
Great! Could you please rename this function to something w/o underscores like `numFormatter`, `expFormatter` etc. ?